run-upgrade-tests.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 set -e
 
+if [ "$1" == "postgresql-file" ]; then
+  eval $(pifpaf --env-prefix INDEXER run postgresql)
+elif [ "$1" == "mysql-ceph" ]; then
+  eval $(pifpaf --env-prefix INDEXER run mysql)
```

Comment:
If you use `eval` here, nothing will stop mysql/postgresql for you.

You'll have to call `pifpaf_stop` IIRC to stop it before you finish this script or the process will kept hanging around.

That's why `pifpaf` was invoked from `tox`.

Reply:
Let me know if new change is better